CHANGELOG.md

```diff
@@ -3,592 +3,117 @@
-
-### Bug fixes
-
-- Avoid considering `PATH` updated when the `export` is commented in the shellrc ([#12043](https://github.com/astral-sh/uv/pull/12043))
-- Fix `uv publish` retry on network failures ([#12041](https://github.com/astral-sh/uv/pull/12041))
-- Use a sized stream in `uv publish` to comply with WSGI PyPI server constraints ([#12111](https://github.com/astral-sh/uv/pull/12111))
-- Fix `uv python install --reinstall` when the version was not previously installed ([#12124](https://github.com/astral-sh/uv/pull/12124))
-
-### Preview features
-
-- Fix `uv_build` invocation ([#12058](https://github.com/astral-sh/uv/pull/12058))
-
-### Documentation
-
-- Quote versions string in `python-versions.md` ([#12112](https://github.com/astral-sh/uv/pull/12112))
-- Fix tool concept page headings ([#12053](https://github.com/astral-sh/uv/pull/12053))
-- Update the `[index].authenticate` docs ([#12102](https://github.com/astral-sh/uv/pull/12102))
-- Update versioning policy ([#11666](https://github.com/astral-sh/uv/pull/11666))
-
-## 0.6.5
-
-### Enhancements
-
-- Allow `--constraints` and `--overrides` in `uvx` ([#10207](https://github.com/astral-sh/uv/pull/10207))
-- Allow overrides in `satisfies` check for `uv tool run` ([#11994](https://github.com/astral-sh/uv/pull/11994))
-- Allow users to set `package = true` on `tool.uv.sources` ([#12014](https://github.com/astral-sh/uv/pull/12014))
-- Add support for Windows legacy scripts via `uv run` ([#11888](https://github.com/astral-sh/uv/pull/11888))
-- Return error when running uvx with a `.py` script ([#11623](https://github.com/astral-sh/uv/pull/11623))
-- Warn user on use of `uvx run` ([#11992](https://github.com/astral-sh/uv/pull/11992))
-
-### Configuration
-
-- Add `NO_BUILD` and `NO_BUILD_PACKAGE` environment variables ([#11968](https://github.com/astral-sh/uv/pull/11968))
-
-### Performance
-
-- Allow overrides in all satisfies checks ([#11995](https://github.com/astral-sh/uv/pull/11995))
-- Respect markers on constraints when validating current environment ([#11976](https://github.com/astral-sh/uv/pull/11976))
-
-### Bug fixes
-
-- Compare major-minor specifiers when filtering interpreters ([#11952](https://github.com/astral-sh/uv/pull/11952))
-- Fix system site packages detection default ([#11956](https://github.com/astral-sh/uv/pull/11956))
-- Invalidate lockfile when empty dependency groups are added or removed ([#12010](https://github.com/astral-sh/uv/pull/12010))
-- Remove prepended sys.path ([#11954](https://github.com/astral-sh/uv/pull/11954))
-- Fix PyPy Python version label ([#11965](https://github.com/astral-sh/uv/pull/11965))
-- Fix error message suggesting `--user` instead of `--username` ([#11947](https://github.com/astral-sh/uv/pull/11947))
+### Breaking changes
 
-### Preview
+- **Update `uv version` to be an interface for project version reads and edits ([#12349](https://github.com/astral-sh/uv/pull/12349))**
```

Comment:
The sentence is a bit hard to parse

```suggestion
- **Update `uv version` to show or change the project version instead of showing the uv version ([#12349](https://github.com/astral-sh/uv/pull/12349))**
```

Reply:
```suggestion
- **Update `uv version` to display or update project versions ([#12349](https://github.com/astral-sh/uv/pull/12349))**
```